src/store/migrations/19_remove_existing_ledger_accounts.ts

```diff
@@ -0,0 +1,25 @@
+import { Account, AccountType } from "../types";
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export const removeExistingLedgerAccounts = {
```

Comment:
Can you add a test for the migration :)
You can copy a wallet state from pre this version and test it

Reply:
@monokh added this way to test the state, also you can run it `npx ts-node ./src/store/migrations/19_remove_existing_ledger_accounts.test.ts` but to be honest I think that can be better if I try later to find a way to test all migrations and don't need to upload any json files or maybe we can set a folder for all states and examples per version.

![Screen Shot 2022-05-25 at 7 27 45 AM](https://user-images.githubusercontent.com/1889891/170261934-ea5e5267-45f8-4306-980b-655773de2fa3.png)